### PR TITLE
Update oraclelinux docs

### PR DESCRIPTION
**Pull request by jlbutler**
* update pointer to new container-images build repo
    * update examples to use 7

Signed-off-by: Jesse Butler <jesse.butler@oracle.com>

### CODE REVIEW

File: oraclelinux/content.md [hunk 1]
```diff
@@ -25,21 +25,15 @@ The Oracle Linux images are intended for use in the **FROM** field of an applica
...
 
-Please review [Chapter 29 of the Oracle Linux 7 Administrator&apos;s Guide](https://docs.oracle.com/cd/E52668_01/E54669/html/ol7-docker.html) for details on how to install or upgrade the Unbreakable Enterprise Kernel and Docker Engine packages.
-
-### Oracle Linux 6
-
-Please review the [additional Linux programs supported under Oracle Linux support](http://docs.oracle.com/cd/E37670_01/E63012/html/ol6-lic-addl-packages.html) chapter for more information on the scope of support for Docker.
-
-Please review [Chapter 10 of the Oracle Linux 6 Administrator&apos;s Solutions Guide](http://docs.oracle.com/cd/E37670_01/E37355/html/ol_docker.html) for details on how to install or upgrade the Unbreakable Enterprise Kernel and Docker Engine packages.
+Please review [Chapter 29 of the Oracle Linux 7 Administrator&apos;s Guide](https://docs.oracle.com/cd/E52668_01/E54669/html/ol7-docker.html) for details on how to install or upgrade the Unbreakable Enterprise Kernel and Oracle Container Runtime for Docker packages.
```
Comment (tianon) [inline oraclelinux/content.md]: This link is a 404 for me -- expected?

**Comment by tianon** [comment]
FYI @Djelibeybi again :smile:

**Comment by tianon** [comment]
(LGTM besides the 404-ing link, which is probably a separate issue entirely)

**Comment by Djelibeybi** [comment]
Wow, that's an old link. Sorry I missed it. Please change that entire paragraph to:

```
Please review the [Oracle Container Runtime for Docker User's Guide](https://docs.oracle.com/cd/E52668_01/E87205/html/index.html) for details on how to install or upgrade the Oracle Container Runtime for Docker on Oracle Linux 7.
```

Thanks!

**Comment by jlbutler** [comment]
Ok thanks @Djelibeybi, all set now. Rebased as well.
PTAL @tianon thanks!

File: oraclelinux/content.md [hunk 2]
```diff
@@ -25,25 +25,19 @@ The Oracle Linux images are intended for use in the **FROM** field of an applica
...
-Please review [Chapter 10 of the Oracle Linux 6 Administrator&apos;s Solutions Guide](http://docs.oracle.com/cd/E37670_01/E37355/html/ol_docker.html) for details on how to install or upgrade the Unbreakable Enterprise Kernel and Docker Engine packages.
+Please review the [Oracle Container Runtime for Docker User's Guide](https://docs.oracle.com/cd/E52668_01/E87205/html/index.html) for details on how to install or upgrade the Oracle Container Runtime for Docker on Oracle Linux 7.
 
 ## Community Support
 
 For Oracle Linux users without a paid support subscription, the following resources are available:
 
 -	The [Oracle Linux Forum](https://community.oracle.com/community/server_%26_storage_systems/linux/oracle_linux) on the [Oracle Technology Network Community](https://community.oracle.com/welcome).
--	The `##oraclelinux` IRC channel on Freenode.
+-	The `#oraclelinux` IRC channel on Freenode.
```
Comment (Djelibeybi) [inline oraclelinux/content.md]: This should actually be #oracle-linux on Freenode.

Comment (jlbutler) [reply oraclelinux/content.md]: Ok, fixed!

**Comment by jlbutler** [comment]
Comment addressed, should be all set now. All good to go @tianon, thanks!

**Pull request merged**